src/tests/MessageTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace NotificationChannels\Signal\Test;
+
+use NotificationChannels\Signal\SignalMessage;
+use PHPUnit\Framework\TestCase;
+
+class MessageTest extends TestCase
+{
+  // @var NotificationChannels\Signal\SignalMessage
+  protected $message;
+
+  // @test
+  protected function setUp()
+  {
+    parent::setUp();
+
+    $this->message = new SignalMessage();
+  }
+
+  public function provides_a_create_method()
+  {
+    $message = SignalMessage::create();
+
+    $this->assertInstanceOf(SignalMessage::class, $message);
+  }
+
+  public function sets_username()
+  {
```

Comment:
Could we get some more in depth tests please?

Reply:
Added test for message.